src/App.js

```diff
@@ -1,25 +1,57 @@
 import React from 'react';
 
+import Clock from './components/Clock';
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+  };
 
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
+  render() {
+    return (
+      <div className="App">
+        <h1>React clock</h1>
+        <div className="App__clockface">
+          <p>
+            Current time:
+            <br />
+            {this.state.isClockVisible && <Clock name={this.state.clockName} />}
+          </p>
+        </div>
+        <div className="App__buttons">
+          <button
+            className="App__button"
+            type="button"
+            onClick={() => {
+              this.setState({ isClockVisible: true });
```

Comment:
It's better to create separate methods in your components and just call them here onClick